sonarts-sq-plugin/sonar-typescript-plugin/src/main/resources/org/sonar/l10n/typescript/rules/typescript/S881.html

```diff
@@ -0,0 +1,36 @@
+<p>The use of increment and decrement operators in method calls or in combination with other arithmetic operators is not recommended, because:</p>
```

Comment:
@vilchik-elena maybe mention that usage in for incrementor is OK?

Reply:
@saberduck it's not function call or arithmetic operator, so I think it should be fine